roles/certificate/README.md

```diff
@@ -66,6 +66,16 @@ For more information about Ansible Galaxy, go to https://galaxy.ansible.com/docs
    zone: 'Business App\\Enterprise CIT'
    EOF
    ```
+
+   **Venafi as a Service EU**:
+   
+   ```sh
+   cat <<EOF >>credentials.yml
+   url: 'https://api.eu.venafi.cloud'
```

Comment:
As Stan pointed put in other MRs, check which url should be used here for EU

Reply:
Done